run_tests.py

```diff
@@ -72,10 +72,17 @@ def load_tests(lines):
 
 def parse_options(checker, options):
     """Parse the CLI-style flags from `options` and expose to `checker`"""
+    extra_kwargs = {}
+    if int(flake8.__version__.split('.')) >= 6:
```

Comment:
I think this could also use `flake8.__version__.count('.')`.

... but I'm not sure what this condition is expressing that indicates the need for `formatter_names`? As written, I worry it's a fragile check.

Reply:
The right check is `if int(flake8.__version__.split('.')[0]) >= 6:`.

... or rather "`if this is a flake8 v6 version, then add the extra "formatter_names" kwarg`"

Reply:
Ah. 🙂 

In that case, let's use `.split('.', 1)[0]`.

Reply:
Great catch! Don't have all my linters fully set-up cross-project 😅 

ETA 10m (but hopefully there's one more CI REPL now, so that I know that there's "nothing more that needs fixing" :pray:)

Reply:
Er, actually, flake8 offers `__version_info__`: https://github.com/PyCQA/flake8/blob/87198e50f145bcc4068ce435939e8635d373ddc0/src/flake8/__init__.py#L21

... which appears to go back a few versions, so that's even better.